src/main/definitions.ts

```diff
@@ -0,0 +1,45 @@
+import { Response } from 'node-fetch';
```

Comment:
Not a practical one, but let's rename this to `types.ts` to match the similar setups in our other repos?

Reply:
Oh yeah, I meant to ask your opinion on it. I thought about the scenario when a user just simply wants to instantiate OAuth2 with `accessToken`, `expiresAt` should be provided along with accessToken otherwise it will try to refresh tokens. Or should we ignore this case and advise them to use just Bearer? 🤔